docs/reference/query-dsl/text-expansion-query.asciidoc

```diff
@@ -87,14 +87,9 @@ is an optimization technique used by {es} to skip documents that cannot score
 competitively against the current best matching documents. However, the tokens
 generated by the ELSER model don't work well with the Max WAND optimization.
 Consequently, enabling Max WAND can actually increase query latency for
-`text_expansion`. For datasets of a significant size, disabling Max
-WAND leads to lower query latencies.
-
-Max WAND is controlled by the
-<<track-total-hits, track_total_hits>> query parameter. Setting track_total_hits
-to true forces {es} to consider all documents, resulting in lower query
-latencies for the `text_expansion` query. However, other {es} queries run slower
-when Max WAND is disabled.
+`text_expansion`.
+Disabling Max WAND by setting the <<track-total-hits, track_total_hits>>
+query parameter to false can improve query performance.
```

Comment:
Two things:

Max WAND is disabled by setting `track_total_hits` to true, which in turn changes the consequences of this language.

I would revert to the language in your previous commit here. It was concise and refrained from making a strong recommendation to set `track_total_hits` to true, which I think is the point of this PR.

This new language is more ambiguous and could lead users to think that setting `track_total_hits` to true can improve query performance in general.

Reply:
Reworded, thanks.